skyrl-train/examples/mini_swe_agent/mini_swe_trainer.py

```diff
@@ -0,0 +1,72 @@
+from typing import List
+import torch
+import math
+from loguru import logger
+
+from skyrl_train.generators.base import (
+    GeneratorInput,
+    GeneratorOutput,
+)
+from skyrl_train.trainer import RayPPOTrainer
+
+
+class MiniSWEPPOTrainer(RayPPOTrainer):
```

Comment:
Why do we need to subclass and modify this? This shouldn't be needed, we should first fix the abstractions before merging the PR.